R/count_teleconnections.R

```diff
@@ -55,6 +55,101 @@ count_watershed_teleconnections <- function(data_dir,
   import_raster(paste0(data_dir, crop_file_path)) ->
     cropcover_USA
 
+      # START GCAM CROP CLASS BINDING
+
+      # Load in the GCAM classes CSV.
+      gcam_csv <- get_crop_mapping()
```

Comment:
The `get_crop_mapping` function needs to be made visible at the package level by adding `@export` in the function docstring in `io.R`